modules/apps/frontend-js/frontend-js-react-web/webpack.config.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+const path = require('path');
+const { container: { ModuleFederationPlugin } } = require('webpack');
+
+module.exports = {
+	context: path.resolve(__dirname),
+	devtool: 'source-map',
+	entry: './null.js',
+	mode: 'development',
```

Comment:
Should mode be set to `development` always? Shouldn't this depend on env flags? Why isn't this whole file inherited as default configs like everything else we do in `liferay-npm-scripts`?

Reply:
> Should mode be set to development always?

Of course not. We should let the user choose.

> Why isn't this whole file inherited as default configs like everything else we do in liferay-npm-scripts?

Well, that's why I sent this PR as draft, because I want to discuss the solution first and, once we like it, move it to `npm-scripts` or `npm-bundler`. I would say `npm-scripts` now and then decide how to make this features available to projects outside `liferay-portal`, but we can follow any other strategy if we find it more useful...

Reply:
> Well, that's why I sent this PR as draft, because I want to discuss the solution first and, once we like it, move it to npm-scripts or npm-bundler. I would say npm-scripts now and then decide how to make this features available to projects outside liferay-portal, but we can follow any other strategy if we find it more useful...

Yes, let's bake this into `liferay-npm-scripts` which is the closer to DXP build and we can iterate from there. I'd rather keep DXP build as close to what we have now as possible rather than having another intermediate step with files all over the place. 

Future strategy should be to also allow for `liferay-npm-scripts` to be usable outside of Liferay's internal build to simplify our build toolchain rather than having to create another offering.

Reply:
Yes, that's what I thought too. Better make `npm-scripts` customer-ready than keep having two different tools. Especially now that we won't be doing any bundling (because webpack will do it).

Reply:
This has been fixed in https://github.com/izaera/liferay-frontend-projects/commits/issue-250